### ISSUE TITLE
system with forcing

### ISSUE THREAD

**Issue by dialuser** [dynamicslab/pysindy#180]
Hi 
I have a system described by
dS/dt = f(S, P, E), it can be as simple as 
e.g., dS/dt = P-E
in which P and E are considered forcing terms. Now, I have observations of S, P, and E, what's the best way to formulate SINDy?
Here's what I have done:
- First, concatenate the 3 time series, S, P, and E
        Xtrain= np.stack([tsS,tsP, tsE], axis=-1)
- Perform minimax scaling, Xtrain = scaler.fit_transform(Xtrain) <- I'm thinking this may help stabilize the estimator
- Define SINDy model
           model = ps.SINDy(
            differentiation_method= differentiator,
            optimizer= stlsq_optimizer,
            feature_library=ps.PolynomialLibrary(degree=1),
            feature_names=["S", "P", "E"],
        )
However, the model identifies three equations in terms of S', P', and E'. Is there a way to tell SINDy to identify a single equation, S', treating the other two as forcing terms?

Thanks,
Alex

**Comment by akaptano** [comment]
You can do this with a little more work, but is it so bad that it generates equations for E' and P'? These are generated independently of the equation for S', so you can safely ignore them without those equations affecting the S' fit.

**Comment by dialuser** [comment]
Hi,
My time series are all "observed" data not synthetic data. So far, only smooth differentiation with a certain order and certain solver seem to produce meaningful results. But I found a control variable feature (https://pysindy.readthedocs.io/en/latest/examples/1_feature_overview.html?highlight=control#sindy-with-control-sindyc) that is better suited to my situation.
